client/components/action-card/README.md

```diff
@@ -78,3 +78,17 @@ Button target, to use in conjunction with `buttonHref`
   - **Default:** `undefined`
 
 Button onClick handler
+
+### `compact`
+  - **Type:** `Boolean`
+  - **Required:** `no`
+  - **Default:** `true`
+
+Determines whether or not this is a compact card
+
+### `illustration`
+  - **Type:** `String`
+  - **Required:** `no`
+  - **Default:** `undefined`
+
+An illustration to show on the card
```

Comment:
I love that you added this to the component. Would it help to be more specific about the `string` to put here? i.e. relative URL, etc.

Reply:
Good point 👍